File: Beecrowd/1159 Soma de Pares Consecutivos/main.py
X = 1

while X != 0:
  X = int(input())
  if(X != 0):
    i = X
    cont = 0
    soma = 0
    while(cont < 5):
      if i % 2 == 0:
        soma += i
        cont += 1
      i += 1     
    print(soma)

File: Beecrowd/1159 Soma de Pares Consecutivos/main.cpp
#include <iostream>
 
using namespace std;

int main(){
  int X, i, soma, cont;

  do{
    cin >> X;
    if(X != 0){
      soma = 0;
      for(i = X, cont = 0; cont < 5; i++){
        if(i % 2 == 0){
          soma += i;
          cont++;
        }
      }
    cout << soma << endl;
    }
  }while(X != 0);
  return 0;
}